code/modules/mob/new_player/preferences_setup.dm

```diff
@@ -273,13 +279,16 @@ datum/preferences
 			if(head_accessory_style && head_accessory_style.species_allowed)
 				var/icon/head_accessory_s = new/icon("icon" = head_accessory_style.icon, "icon_state" = "[head_accessory_style.icon_state]_s")
 				head_accessory_s.Blend(rgb(r_headacc, g_headacc, b_headacc), ICON_ADD)
-				eyes_s.Blend(head_accessory_s, ICON_OVERLAY)
+				face_s.Blend(head_accessory_s, ICON_OVERLAY)
 
 		var/datum/sprite_accessory/facial_hair_style = facial_hair_styles_list[f_style]
 		if(facial_hair_style && facial_hair_style.species_allowed)
 			var/icon/facial_s = new/icon("icon" = facial_hair_style.icon, "icon_state" = "[facial_hair_style.icon_state]_s")
-			facial_s.Blend(rgb(r_facial, g_facial, b_facial), ICON_ADD)
-			eyes_s.Blend(facial_s, ICON_OVERLAY)
+			if(current_species.name == "Slime People") // whee I am part of the problem
+				facial_s.Blend(rgb(r_skin, g_skin, b_skin,160), ICON_ADD)

```

Comment:
Should probably put a space before that 160 just for consistency (see update_icons.dm lines 466 and 503)